packages/worker/src/worker.ts

```diff
@@ -658,10 +658,13 @@ export class Worker {
   }
 
   /**
-   * Start shutting down the Worker. Immediately transitions {@link state} to `'STOPPING'` and asks Core to shut down.
-   * Once Core has confirmed that it's shutting down, the Worker enters `'DRAINING'` state unless the Worker has already
-   * been `'DRAINED'`. All currently running Activities are Cancelled. Once all currently running Activities and
-   * Workflow Tasks have completed, the Worker transitions to `'STOPPED'`.
+   * Start shutting down the Worker. The Worker stops polling for new tasks and sends
+   * {@link https://typescript.temporal.io/api/namespaces/activity#cancellation | cancellation} to running Activities.
```

Comment:
I'd mention that these activities don't end up cancelled, they end up failed in case cancellation is respected.

Reply:
Also might want to note that we the reason for cancellation in this case will be WORKER_SHUTDOWN (please double check the exact reason).

Reply:
Checked. How's this?

The Worker stops polling for new tasks and sends
   * {@link https://typescript.temporal.io/api/namespaces/activity#cancellation | cancellation} (via a
   * {@link CancelledFailure} with `message` set to `'WORKER_SHUTDOWN'`) to running Activities. Note: if the Activity
   * accepts cancellation (i.e. re-throws or allows the `CancelledFailure` to be thrown out of the Activity function),
   * the Activity Task will be marked as failed, not cancelled. It's helpful for the Activity Task to be marked failed
   * during shutdown because the Server will retry the Activity sooner (than if the Server had to wait for the Activity
   * Task to time out).

Reply:
LGTM